pkg/binary/envoy/fetch.go

```diff
@@ -87,9 +86,7 @@ func (r *Runtime) BinaryStore() string {
 }
 
 func (r *Runtime) platformDirectory(key *manifest.Key) string {
-	platform := strings.ToLower(key.Platform)
-	platform = strings.ReplaceAll(platform, "-", "_")
-	return filepath.Join(r.BinaryStore(), key.Flavor, key.Version, platform)
+	return filepath.Join(r.BinaryStore(), key.Flavor, key.Version, key.Platform)
```

Comment:
just want to confirm that this function is covered by `TestRuntime_Fetch` in test?

Reply:
it isn't needed anymore because we are no longer creating invalid platform keys. (tests of conversion ending up in the correct place are elsewhere in locate_test)